README.md

```diff
@@ -126,13 +113,10 @@ require("direnv-nvim").setup({
 require("lspconfig").lua_ls.setup({ autostart = false })
 require("lspconfig").clangd.setup({ autostart = false })
 
--- Start lsp only after direnv.nvim is finished
-vim.api.nvim_create_autocmd("User", {
-	pattern = { "DirenvLoaded", "DirenvNotFound" }, -- this example starts the lsp when the direnv was loaded or when there is no .envrc found
-	callback = function()
+-- Start the LSP on direnv changes for the given filetypes we have defined setup for
+M.on_direnv_finished({ filetype = {"lua", "c"} }, function ()
```

Comment:
I would leave the `nvim_create_autocmd` written out without using `on_direnv_finished` as it illustrates more clearly what is being triggered.

Like this:
```
vim.api.nvim_create_autocmd("User", {
    pattern = { "DirenvReady", "DirenvNotFound", "DirenvBlocked" },
    group = "direnv-nvim",
    callback = function()
        vim.cmd("LspStart")
    end,
})
```

The `on_direnv_finished` example could be added but commented out? As you feel

I personally did not end up using the utility as I start the lsp on all filetypes.

PS: `M` should probably be replaced with `require("direnv-nvim")` ?